src/civetweb.c

```diff
@@ -17314,7 +17314,7 @@ websocket_client_thread(void *data)
 
 	void *user_thread_ptr = NULL;
 
-#if !defined(_WIN32)
+#if !defined(__ZEPHYR__)
```

Comment:
```suggestion
#if !defined(_WIN3) && !defined(__ZEPHYR__)
```

Reply:
I agree with it. Will be changed with the next push next week. :)

Reply:
Updated preprocessor's if